api/authenticator.py

```diff
@@ -425,12 +426,14 @@ def set_authorization_identifier(self, authorization_identifier):
         self.authorization_identifier = authorization_identifier
         self.authorization_identifiers = authorization_identifiers
 
-class CirculationPatronProfileStorage(PatronProfileStorage):
+class CirculationPatronProfileStorage(PatronProfileStorage, LibraryAnnotator):
```

Comment:
This works, but I think it's more complex than necessary, and I'm a little worried about it because `LibraryAnnotator` was only designed to be used to make OPDS feeds. What happens if you use the normal `url_for` from the `flask` package instead of `self.url_for`?

Reply:
I had tried the normal `url_for` originally, but it throws an error message--something about taking the wrong number of arguments, I think?  I will take another look, though!